context/brand-context/default/scss/30-mixins/focus.scss

```diff
@@ -1,6 +1,8 @@
-@mixin u-focus-outline {
+@mixin u-focus-outline($reflow: false) {
 	&:focus {
 		outline: 3px solid $context--focus-color;
-		-webkit-transform: translateZ(0); // reflow needed to ensure outline styles visible in Safari v13+
+		@if $reflow == true {
+			-webkit-transform: translateZ(0); // reflow needed to ensure outline styles visible in Safari v13+
```

Comment:
Is there a negative effect of applying this all browsers? i.e. just using `transform: translateZ(0)`

Reply:
Replaced the `transform: translateZ(0)` hack with `will-change: transform` to create new layer. 

Removed the parameter and applied it to all browsers as I couldn't see significant effects on the page when tabbing through the page; the tabbing isn't a smooth transition that can be noticed.